app/src/main/java/com/ivy/wallet/logic/csv/CSVMapper.kt

```diff
@@ -169,4 +172,60 @@ class CSVMapper {
         account = 9,
         description = 10
     )
+
+    @ExperimentalStdlibApi
+    private fun financisto() = RowMapping(
+        date = 0,
+        timeOnly = 1,
+        account = 2,
+        amount = 3,
+        transferAmount = 3,
+        accountCurrency = 4,
+        // original currency amount = 5
+        // original currency = 6
+        category = 7,
+        // parent transaction = 8
+        title = 9,
+        type = 10,
+        // project = 11
+        description = 12,
+
+        transformAmount = { amount, transactionType ->
+            if (amount > 0 && transactionType == TransactionType.EXPENSE) {
```

Comment:
`amount > 0 && transactionType == TransactionType.EXPENSE` this check makes no sense. Can't we just check if amount < 0?